ppr/doc/libpaf-ppr.3

```diff
@@ -0,0 +1,120 @@
+.\" Copyright IBM Corp. 2014
+.\"
+.\" The MIT License (MIT)
+.\"
+.\" Permission is hereby granted, free of charge, to any person obtaining a copy
+.\" of this software and associated documentation files (the "Software"), to
+.\" deal in the Software without restriction, including without limitation the
+.\" rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+.\" sell copies of the Software, and to permit persons to whom the Software is
+.\" furnished to do so, subject to the following conditions:
+.\"
+.\" The above copyright notice and this permission notice shall be included in
+.\" all copies or substantial portions of the Software.
+.\"
+.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+.\" AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+.\" LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+.\" FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+.\" IN THE SOFTWARE.
+.\"
+.\" Contributors:
+.\"     IBM Corporation, Adhemerval Zanella - Initial implementation.
+.\"     IBM Corporation, Rajalakshmi S - Initial implementation.
+.TH LIBPAF 2014-04-13 "Linux" "Program Priority Register"
+.SH NAME
+The Power Architecture Facilities - Program Priority Register
+.SH SYNOPSIS
+.sp
+.BI "#include <paf/ppr.h>
+.sp
+.BI "void paf_yield(void);
+.sp
+.BI "void paf_mdoio(void);
+.sp
+.BI "void paf_mdoom(void);
+.sp
+.BI "void paf_ppr_very_low(void);
+.sp
+.BI "void paf_ppr_low(void);
+.sp
+.BI "void paf_ppr_med_low(void);
+.sp
+.BI "void paf_ppr_med(void);
+.sp
+.BI "void paf_ppr_med_high(void);
+.sp
+
+.SH DESCRIPTION
+
+The Program Priority Register header ppr.h provides a mechanism to adjust the
+thread priorities based on lock acquisition, wait and release
+available on Power Architecture.
+
+.PP
+The function
+.BR paf_yield()
+Provide a hint that performance will probably be improved if shared resources
```

Comment:
I think it would be better to continue the sentence with lowercase, e.g.:

The function
.BR paf_yield ()
provides